backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Tests/DelegationsControllerTest.cs

```diff
@@ -115,6 +115,49 @@ public async Task Post_DeleteRules_Success()
             AssertionUtil.AssertEqual(expected, actual);
         }
 
+        /// <summary>
+        /// Test case: Calling the POST operation for DeleteRules to perform a valid deletion of resourceRegistryId
+        /// Expected: DeleteRules returns status code 201 and list of rules created match expected
+        /// 
+        /// Scenario:
+        /// Calling the POST operation for DeleteRules to perform a valid deletion
+        /// Input:
+        /// List of two one rule in one policy for deletion of the resource between for a single offeredby/coveredby combination resulting in a single policyfile beeing updated.
+        /// Expected Result:
+        /// Rules are deleted and returned with the CreatedSuccessfully flag set and rule ids
+        /// Success Criteria:
+        /// DeleteRules returns status code 201 and list of rules deleted to match expected
+        /// </summary>
+        [Fact]
+        public async Task Post_DeleteResourceRegistryRules_Success()
+        {
+            // Arrange
+            Stream dataStream = File.OpenRead("Data/Json/DeleteRules/ReadResource2_50001337_20001337.json");
+            StreamContent content = new StreamContent(dataStream);
+            content.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+
+            List<Rule> expected = new List<Rule>
+            {
+                TestDataUtil.GetRuleModel(20001336, 50001337, "20001337", AltinnXacmlConstants.MatchAttributeIdentifiers.UserAttribute, "read", null, null, createdSuccessfully: true, resourceRegistryId: "resource2"),
```

Comment:
createdSuccessfully is irrelevant for a delete operation. 
Should set the expected ruleIds which are to be deleted for verification against the response